CMakeLists.txt

```
@@ -20,6 +22,20 @@ if(APPLE)
     OPTION(APPLE_OUTPUT_DYLIB "Output dylib shared library" OFF)
 endif(APPLE)
 
+if(__OPENCL_PYTHON_PACKAGE)
+  set(__OPENCL_PYTHON_PACKAGE ON CACHE BOOL "" FORCE)
+  set(USE_GPU OFF CACHE BOOL "" FORCE)
+  set(USE_MPI OFF CACHE BOOL "")
+  set(USE_OPENMP ON CACHE BOOL "")
+  set(USE_SWIG OFF CACHE BOOL "")
+  set(USE_HDFS OFF CACHE BOOL "")
+  set(USE_R35 OFF CACHE BOOL "")
+  set(USE_TIMETAG OFF CACHE BOOL "")
+  set(USE_DEBUG OFF CACHE BOOL "")
+  set(BUILD_FOR_R OFF CACHE BOOL "")
+  message(WARNING "\nWARNING: building integrated OpenCL enabled components intended for Python package")
```

I don't believe this `\n` is necessary. `CMake` will already make sure each message is printed on a new line
```suggestion
  message(WARNING "WARNING: building integrated OpenCL enabled components intended for Python package")
```

---

Removed.